bundles/http/Citrix/f1c0fcac-0764-4a21-96b8-77a0316e337d/i18n/de.json

```diff
@@ -1,364 +1,364 @@
-    "83d5ee3a-72fb-4517-975c-53c038e03acf" : "Incident",
-    "150e9d4d-cdd9-4ce9-b684-efa783f92dec" : "Incidentstatus geändert",
-    "369381e8-8e91-4643-895f-07adcbb0ed44" : "Zugewiesen zu",
-    "47221af1-3de4-4d2b-8251-c67a9734ebc6" : "Übergeordnete Aufgabe",
-    "4f7035f0-fd95-45cb-b923-77c3e2490e63" : "Prioritätsdetails",
-    "4e8bfcfb-2221-4032-9bfa-018bcb4e1b6a" : "Incidentstatusänderung (Anrufer)",
-    "cdb9bf0d-2a22-4193-86ac-39fa39200a2f" : "Senden",
-    "805ba22d-4afd-4789-9bc3-b79042dd4ef2" : "Nummer",
-    "bd694f51-e38b-4bbd-9427-54cf2d25e043" : "sys_id",
-    "546ae764-4372-4efe-9f99-b34ffd25a96c" : "Es wurden keine Incidents gefunden.",
-    "7ec7dfa6-2b39-47c7-aabe-3e66f96077f1" : "Incidentzuweisungsänderung (geöffnet von)",
-    "87fa4223-6ef1-48ee-b9d5-1f57b1e2f29a" : "Zuweisungsgruppe",
-    "505ac148-d73d-401b-ad8b-66d2decd607d" : "Genehmigung",
-    "4d9fd23c-98d4-483b-9266-0e435008a8dd" : "Text",
-    "7c24f383-13cb-4901-9122-04aa40837b02" : "Incidentstatus geändert",
-    "aa9646e7-855b-4332-b1b1-8833cfef5276" : "{{number}} - '{{short_description}}'",
-    "9249e64b-9fee-4a9b-8e79-e9b4a1fad996" : "Der Incident {{number}} '{{short_description}}' wurde Ihnen zugewiesen.",
-    "3c94ccfa-6b03-4eec-86af-c6815c879db4" : "Meine offenen Incidents",
-    "ba4ee4b1-5e7e-4693-8ad3-8ae62fae8990" : "Noch kein Kommentar hinzugefügt.",
-    "0b18f46d-446c-4e12-9f1a-5eb9a269246a" : "Incidentstatus geändert",
-    "612e944c-d34b-4554-81d6-599b97009bef" : "Zuweisungsgruppe suchen...",
-    "905ea457-c65f-4ccf-a237-f38c5ea3073a" : "Incident zugewiesen",
-    "da1da44a-1a11-41c0-9b79-f7298cb0f87f" : "Abschlussnotizen",
-    "ff09c72f-275d-4976-9bfc-8d6bae7328b5" : "Ort",
-    "22efc99f-db5d-4198-856b-da7e321b3b5e" : "Abschlusscode",
-    "975d050b-86d4-4f29-9fcd-f7990d54a299" : "Incidentzuweisungsänderung (Anrufer)",
-    "a7c6b0a8-285f-47ed-b018-6a3c3123fbca" : "Übergeordneter Incident",
-    "eeebc601-6879-44dd-a0e4-628e5d6dcbac" : "Incidents",
-    "f9d0efb1-dc49-470c-b252-224d2f874142" : "Incidentstatusänderung (Zuweisung)",
-    "848d342b-3e4f-451f-a0de-509711fdcc0e" : "Untergeordnete Incidents",
-    "9330feef-358f-4b1c-a53a-dbf594feb6bb" : "Incident aktualisieren",
-    "6d338ca9-cf36-49a8-a61b-07ca91eb05e9" : "Dringlichkeit",
-    "12eeb9f5-2bae-406b-aa45-12ac6667972f" : "Anrufer"
+  "5805e29b-56fb-42c1-944d-867c91adaaca": {
+    "fd5de183-6fc6-4b74-8809-2fb122ef71fb": "Auswirkung",
+    "43e9a327-51ba-4856-85cd-fbae6a2d01fc": "Status",
+    "6ca38495-4465-49f2-a2f8-f99835f6b3b3": "Geöffnet am",
+    "724fb4fb-fd21-426a-ba2f-01a38436cd83": "Details",
+    "58fde4f6-7a95-44f7-af99-bb040638211a": "Der Incident {{number}} '{{short_description}}' wurde Ihnen zugewiesen.",
+    "0515b96f-9e7b-449f-840b-263080c4e597": "Kommentar hinzufügen",
+    "1382a579-1fd6-47cd-ac0b-7d89579b5c46": "Incident {{number}} - '{{short_description}}' has been assigned to {{assigned_to__incident.name}}.",
```

Comment:
the {{assigned_to__incident.name}} has been modified,it should be {{assigned_to_incident.name}}